cypress/features/regression/showEditPod.feature

```diff
@@ -1,138 +1,135 @@
     Examples:
-      | cancelText                   |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | cancelText                   | nameOfObject               |
+      | mp150ú¿¡üßä                  | cancelTextOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | cancelTextSpecialCharacter |
 
   @positive
   Scenario Outline: Enable cancel checkbox for a Show Edit Pod component
+    Given I open default "ShowEditPod" component in noIFrame with "showEditPod" json from "commonComponents" using "cancel" object name
     When I click edit Show Edit Pod component
     Then Show Edit Pod component cancel button has color "<color>" and borderColor "<color>"
     Examples:
       | color           |
       | rgb(0, 129, 93) |
 
   @positive
-  Scenario: Enable and disable cancel checkbox for a Show Edit Pod component
-    Given I check cancel checkbox
-    When I uncheck cancel checkbox
+  Scenario: Disable cancel checkbox for a Show Edit Pod component
+    Given I open default "ShowEditPod" component in noIFrame with "showEditPod" json from "commonComponents" using "cancelFalse" object name
       And I click edit Show Edit Pod component
     Then Show Edit Pod component hasn't a cancel button
 
   @positive
   Scenario Outline: Change Show Edit Pod deleteText to <deleteText>
-    When I set deleteText to <deleteText> word
+    Given I open default "ShowEditPod" component in noIFrame with "showEditPod" json from "commonComponents" using "<nameOfObject>" object name
       And I click edit Show Edit Pod component
     Then Show Edit Pod deleteText on preview is set to <deleteText>
     Examples:
-      | deleteText                   |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | deleteText                   | nameOfObject               |
+      | mp150ú¿¡üßä                  | deleteTextOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | deleteTextSpecialCharacter |
 
   @positive
   Scenario Outline: Change Show Edit Pod saveText to <saveText>
-    When I set saveText to <saveText> word
+    When I open default "ShowEditPod" component in noIFrame with "showEditPod" json from "commonComponents" using "<nameOfObject>" object name
       And I click edit Show Edit Pod component
     Then Show Edit Pod saveText on preview is set to <saveText>
     Examples:
-      | saveText                     |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | saveText                     | nameOfObject             |
+      | mp150ú¿¡üßä                  | saveTextOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | saveTextSpecialCharacter |
 
   @positive
   Scenario: Enable saving checkbox for a Show Edit Pod component
-    When I check saving checkbox
+    When I open default "ShowEditPod" component in noIFrame with "showEditPod" json from "commonComponents" using "save" object name
```

Comment:
`Given` - `When` - `Then`

Reply:
I won't change it. Important is step `When` and it specifies step `Then`